src/index.html

```diff
@@ -11,12 +12,110 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
```

Comment:
The `crossorigin` attribute should have a value of either `anonymous` or `use-credentials`. The current value `attr-req-value` is incorrect and will not work as intended. Please update it to `anonymous` if you do not need credentials, or `use-credentials` if you do.